homeassistant/components/glances/sensor.py

```diff
@@ -117,11 +333,15 @@ def __init__(
         self.unsub_update = None
 
         self.entity_description = description
-        self._attr_name = f"{name} {sensor_name_prefix} {description.name_suffix}"
+        self._attr_name = f"{sensor_name_prefix} {description.name_suffix}"
+        # if name exists preserve previous naming
+        if name:
```

Comment:
We don't need this case if we've already released this integration with unique ids for the sensor entities. Entities with a unique id are protected against changes to the entity id and the user can customize the name.